.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,11 +1,12 @@
 ---
-name: Feature request
+name: Core Feature request
 about: Suggest an idea for this project
-title: "[Feature]"
+title: "[Core Feature]"
 labels: enhancement, untriaged
 assignees: ''
 
 ---
+**[ ] Is this a blocker for you to adopt Flyte**
```

Comment:
can we mark this optional too? i feel like a lot of our issues are filed by the existing community so this question isn't really applicable/relevant